Makefile

```diff
@@ -20,16 +33,16 @@ clean-up:
 
 test: | gofmt vet lint
 	go mod tidy
-	go test ./internal/... -cover
+	go test $(TEST_PKGS) $(TEST_FLAGS)
```

Comment:
nit : @johnweldon , Making makefile generic is good but since this makefile is only for this project what would be the additional benefit of picking the packages from env var ? 
I believe the previous change was better.

Reply:
This is actually to make it easier to iterate quickly in development - I can run `TEST_PKGS="./internal/resources/client/integration/..." make test` to run only the tests I'm working on more quickly.  The default values are the same as they were before, but just allows overriding the test behaviour without modifying the makefile.  I'm happy to revert it, it just makes it easier for development.